packages/payment-integration/src/errors/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as CustomError } from './CustomError';
+export { default as EmbeddedCheckoutUnsupportedError } from './EmbeddedCheckoutUnsupportedError';
```

Comment:
EOL

Reply:
Argh, got to properly set up my editor on my new laptop hehe